ext/drmgr/drmgr.h

```diff
@@ -489,6 +489,30 @@ DR_EXPORT
 bool
 drmgr_is_last_instr(void *drcontext, instr_t *instr);
 
+DR_EXPORT
+/**
+ * Returns the number of instructions (including meta-instructions) inside a basic block
+ * \p bb.
+ *
+ * The function iterates over the bb in order to obtain the count. The result is not
+ * cached. Therefore, avoid using this function during the insert stage of the
+ * instrumentation process.
+ */
+size_t
+drmgr_get_bb_size(instrlist_t *bb);
```

Comment:
These functions could apply to a trace too, right?  Maybe they shouldn't be oriented toward a "bb"?

Reply:
Done